packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -1206,3 +1206,10 @@ export function isFargateCompatible(compatibility: Compatibility): boolean {
 export function isExternalCompatible(compatibility: Compatibility): boolean {
   return [Compatibility.EXTERNAL].includes(compatibility);
 }
+
+/**
+ * Return true for windows instance
+ */
+export function isWindowsInstance(operatingSystemFamily?: string): boolean | undefined {
+  return operatingSystemFamily?.startsWith('WINDOWS_SERVER_');
```

Comment:
thanks for making a helper for this!

looks like line 445 above is also checking for WINDOWS - maybe we should make that line use the same helper also? (`.includes('WINDOWS')` might be more generic 

the name `isWindowsInstance` is slightly odd here as there are no "instances" in the task definition world - I think 
`isWindowsOS` is probably better. 

BTW - what do you think about adding a helper on the `OperatingSystemFamily` class directly? I'm not sure what's the convention here, since both places have `operatingSystemFamily` instances, I'm curious if there is a way to allow us to simply write `runtimePlatform.operatingSystemFamily.isWindows()` in both lines.

Reply:
ya changed isWindowsInstance to isWindowsOs.
Added same logic for line 445 as well.

and changed code which can use by Ec2 customers as well where Windows on EC2  only memoryLimitMib is provided,